forward/ft_senstype.m

```diff
@@ -500,7 +500,7 @@
     case 'egi'
       type = any(strcmp(type, {'egi' 'egi32' 'egi64' 'egi128' 'egi256'}));
     case 'meg'
-      type = any(strcmp(type, {'meg' 'ctf' 'ctf64' 'ctf151' 'ctf275' 'ctf151_planar' 'ctf275_planar' 'neuromag' 'neuromag122' 'neuromag306' 'neuromag306_combined' 'bti' 'bti148' 'bti148_planar' 'bti248' 'bti248_planar' 'bti248grad' 'bti248grad_planar' 'yokogawa' 'yokogawa9' 'yokogawa160' 'yokogawa160_planar' 'yokogawa64' 'yokogawa64_planar' 'yokogawa440' 'itab' 'itab28' 'itab153' 'itab153_planar' 'babysquid' 'babysquid74' 'artenis123' 'magview'}));
+      type = any(strcmp(type, {'meg' 'ctf' 'ctf64' 'ctf151' 'ctf275' 'ctf151_planar' 'ctf275_planar' 'neuromag' 'neuromag122' 'neuromag306' 'neuromag306_combined' 'bti' 'bti148' 'bti148_planar' 'bti248' 'bti248_planar' 'bti248grad' 'bti248grad_planar' 'yokogawa' 'yokogawa9' 'yokogawa160' 'yokogawa160_planar' 'yokogawa64' 'yokogawa64_planar' 'yokogawa440' 'itab' 'itab28' 'itab153' 'itab153_planar' 'babysquid' 'babysquid74' 'artenis123' 'magview' 'York 4d'}));
```

Comment:
it should be a single string, no spaces, no capitals. Would `bti248` not be better to describe the sensor array? In that case the template layouts and other 4d specific handling will automatically apply.

We could also make it `yorkinstruments248` if you consider the system sufficiently different from the `bti248`.

Reply:
I have chosen yorkinstruments248 as 4d system uses (I believe) ALS coordinate system while York Instruments switched to RAS (albeit for the same physical sensors). 
ft_senstype identifies the YI sensors as bti248 based on having the same set of channel names, but this is not problematic as the sensor types and units are the same for both systems.